src/main/java/com/farion/onlinebookstore/util/ConditionHolder.java

```diff
@@ -0,0 +1,14 @@
+package com.farion.onlinebookstore.util;
+
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.stereotype.Component;
+
+@Setter
+@Getter
+@Component
+public class ConditionHolder {
+    private Set<Long> condition = new HashSet<>();
+}
```

Comment:
I added this class so that I could dynamically pass a Set of categoru IDs to the validator during the runtime. So, if I add a new category to the database, the validator will only pass valid IDs when creating a new book

Reply:
So this is like sort of a cache, in which you store all category ids which exist in your db, and you check it before saving a book? As far as I understood you are ignoring if some of those ids are not valid, but it is wrong because we should not do that implicitly. The best practice in working with a user - is that you should never decide things for him, and do everything as explicitly as you can, because you can think that some things are obvious for everyone, but usually they are not 😅 
So let's just throw ex if any of the category id is not valid, because user should know if something went wrong, rather then finding out that some of the categories that he chose were ignored and not saved without no apparent reason,

And also We do not need that class as a cache for it to,